index.html

```diff
@@ -6,67 +6,141 @@
-            <li>Gitlab</li>
-            <li>Terminal</li>
-          </ul>
+        <div class="proof proof-space">
+          <img class="project-img" src="img/fola3.PNG" alt="Project 1">
+          <h2>YOU NEED SOLAR</h2>
+          <p>A website created for a digital marketing agency to help improve sales and give information about company's 
+            product. This website also helps the client to contact the company for a business deal and showcases the company's 
+            different services and packages. 
+          </p>
+          <div class="tools">
+            <ul>
+              <li class="line">CSS</li>
+              <li class="line">html</li>
+              <li class="line">Bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+          </div>
+          <a class="project-link" href="http://youneedsolar.com/">See Project</a>
         </div>
-      </div>
-    </section>
-    <div class="socials">
-      <ul>
-        <li><a href="#"><i class="fab fa-github"></i></a></li>
-        <li><a href="#"><i class="fab fa-linkedin-in"></i></a></li>
-        <li><a href="#"><i class="fab fa-twitter"></i></a></li>
-        <li><a href="#"><i class="fab fa-medium-m"></i></a></li>
-      </ul>
-    </div>
-    <div class="point-down">
-      <i class="fas fa-arrow-down"></i>
-      <p class="work">work</p>
-    </div>
-  </main>
+        <div class="proof proof-space">
+          <img class="project-img" src="img/fola3.PNG" alt="Project 1">
+          <h2>YOU NEED SOLAR</h2>
+          <p>A website created for a digital marketing agency to help improve sales and give information about company's 
+            product. This website also helps the client to contact the company for a business deal and showcases the company's 
+            different services and packages. 
+          </p>
+          <div class="tools">
+            <ul>
+              <li class="line">CSS</li>
+              <li class="line">html</li>
+              <li class="line">Bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+          </div>
+          <a class="project-link" href="http://youneedsolar.com/">See Project</a>
+        </div>
+        <div class="proof proof-space">
+          <img class="project-img" src="img/spike.PNG" alt="Project 1">
+          <h2>SPIKE SECURITIES</h2>
+          <p>A website created for a digital marketing agency to help improve sales and give information about company's 
+            product. This website also helps the client to contact the company for a business deal and showcases the company's 
+            different services and packages. 
+          </p>
+          <div class="tools">
+            <ul>
+              <li class="line">CSS</li>
+              <li class="line">html</li>
+              <li class="line">Bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+          </div>
+          <a class="project-link" href="https://spikesecurities.com/">See Project</a>
+        </div>
```

Comment:
- [ optional ] It will be nice to wrap all our projects in a container and give the container the appropriate style.